pyproject.toml

```diff
@@ -75,6 +75,8 @@ wrapt = '*'
 psutil = '*'
 requests = { version = ">=2.26.0", optional = true }
 kafka-python = { version = "*", optional = true }
+uvloop = "^0.17.0"
```

Comment:
If its an experimental feature, probably its a good idea to expose an extras flag for the functionality. Otherwise, it becomes a mandatory package. Also in your code imports uvloop even if not enabled.